src/js/tracks/text-track.js

```diff
@@ -331,7 +335,7 @@ class TextTrack extends Track {
   addCue(originalCue) {
     let cue = originalCue;
 
-    if (!(originalCue instanceof window.vttjs.VTTCue)) {
+    if (window.vttjs && !(originalCue instanceof window.vttjs.VTTCue)) {
```

Comment:
I was mocking vttjs out https://github.com/videojs/video.js/pull/4146/files#diff-3b4ce30c38112a433d137fb9bf1bcbc9R21 but this probably is a better choice.